### DISCUSSION TITLE
Welcome to FinRL-Trading Discussions!

### DISCUSSION THREAD

**Discussion by YangletLiu** [AI4Finance-LLC/Deep-Reinforcement-Learning-for-Automated-Stock-Trading-Ensemble-Strategy-ICAIF-2020#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by omar94khan** [comment]
Hi Xiao!

I am trying to replicate your work so that I can experiment with different RL Algorithms and compare their performances on the environment.

I noticed that you had referred to a file "sp500_fundamental_199601_202210.csv" in your preprocessing steps to prepare the data. This dataset, according to your notes, can be downloaded via WRDS - a portal I do not have access to.

Would it be possible for you to share the referred file with me so that I can continue with the experimentation.

Regards,
Omar
